en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -993,6 +977,9 @@ You can configure your directory structure like so:
 }
 ```
 
+> [!WARNING]
+> Using `@silverstripe/webpack-config` will keep your transpiled bundle size smaller and ensure you are using the correct versions of `react-apollo` and `graphql-tag`, as these will automatically be added as [webpack externals](https://webpack.js.org/configuration/externals/). If you are not using that npm package, it is very important you use the correct versions of those dependencies.
+
```

Comment:
Moved this from above to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.